### PR TITLE
Change kube-up.sh to use Containerd as default runtime

### PR DESCRIPTION
**Pull request by sonyafenge**
**What type of PR is this?**

> /kind feature

**What this PR does / why we need it**:
1. Change kube-up.sh to use containerd as default runtime
2. add docker and containerd installation for ubuntu 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cluster/gce/config-common.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # Copyright 2016 The Kubernetes Authors.
+# Copyright 2020 Authors of Arktos - file modified.
```
Comment (yb01) [inline cluster/gce/config-common.sh:4]: this is likely a glitch in the tool to update the lic headers. there is no changes in the file in the PR. so please remove this file.

Comment (sonyafenge) [reply cluster/gce/config-common.sh]: just removed this file.

File: cluster/gce/config-default.sh [hunk 2]
```diff
@@ -138,17 +138,27 @@ SCALEOUTSERVER_IMAGE=${SCALEOUTSERVER_IMAGE:-${MASTER_IMAGE}}
...
+CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-containerd}
+# Set default values with override
+if [[ "${CONTAINER_RUNTIME}" == "docker" ]]; then
+  export CONTAINER_RUNTIME_ENDPOINT=${KUBE_CONTAINER_RUNTIME_ENDPOINT:-unix:///var/run/dockershim.sock}
+  export CONTAINER_RUNTIME_NAME=${KUBE_CONTAINER_RUNTIME_NAME:-docker}
+  export LOAD_IMAGE_COMMAND=${KUBE_LOAD_IMAGE_COMMAND:-}
+elif [[ "${CONTAINER_RUNTIME}" == "containerd" ]]; then
+  export CONTAINER_RUNTIME_ENDPOINT=${KUBE_CONTAINER_RUNTIME_ENDPOINT:-unix:///run/containerd/containerd.sock}
+  export CONTAINER_RUNTIME_NAME=${KUBE_CONTAINER_RUNTIME_NAME:-containerd}
+  #export LOG_DUMP_SYSTEMD_SERVICES=${LOG_DUMP_SYSTEMD_SERVICES:-containerd}
```
Comment (yb01) [inline cluster/gce/config-default.sh:150]: please remove the unused code.

File: cluster/gce/gci/configure.sh [hunk 3]
```diff
@@ -759,6 +759,128 @@ EOF
...
+  # bailout if we are not on ubuntu
+  echo "Installing docker .."
+  if ! command -v apt-get >/dev/null 2>&1; then
+    echo "Unable to automatically install docker. Bailing out..."
+    return
+  fi
+  # Install Docker deps, some of these are already installed in the image but
+  # that's fine since they won't re-install and we can reuse the code below
+  # for another image someday.
+  apt-get update
```
Comment (yb01) [inline cluster/gce/gci/configure.sh:773]: should all those root level calls with a "sudo " ?

File: cluster/gce/gci/configure.sh [hunk 4]
```diff
@@ -769,7 +891,7 @@ set-broken-motd
 KUBE_HOME="/home/kubernetes"
 KUBE_BIN="${KUBE_HOME}/bin"
 
-ensure-docker
+ensure-container-runtime
```
Comment (yb01) [inline cluster/gce/gci/configure.sh:894]: please verify this does run on minion nodes.

**Review by yb01 — APPROVED** [review]
/lgtm

File: cluster/gce/gci/configure.sh (hunk 4 above)
Comment (sonyafenge) [reply cluster/gce/gci/configure.sh:894]: yes, both minion nodes and master call this "configure.sh" for installation service.

File: cluster/gce/config-default.sh (hunk 2 above)
Comment (sonyafenge) [reply cluster/gce/config-default.sh]: removed

**Pull request merged**